.cron.yml

```diff
@@ -32,3 +32,9 @@ jobs:
           treeherder-symbol: bump-ac
           target-tasks-method: bump_android_components
       when: [{hour: 14, minute: 0}]
+    - name: ui-test-screenshots-x86
+      job:
+          type: decision-task
```

Comment:
@AaronMT not sure if this has to be the type...

Reply:
Not sure (@rpappalax ?)